azureproject/settings.py

```diff
@@ -93,3 +93,67 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+if 'RUNNING_IN_PRODUCTION' in os.environ:
+    # Configure allowed host names that can be served and trusted origins for Azure Container Apps.
+    ALLOWED_HOSTS = ['.azurecontainerapps.io']
+    CSRF_TRUSTED_ORIGINS = ['https://*.azurecontainerapps.io']
+    DEBUG = False
+    DEBUG_PROPAGATE_EXCEPTIONS = True
```

Comment:
Why is this True?

https://docs.djangoproject.com/en/5.0/ref/settings/#debug-propagate-exceptions

I think it's typically False in production?

Reply:
Truthfully -- I have no idea. It was set that way in production.py in the main branch. You're right, it certainly doesn't appear that it should ever be set in production. I did see a comment in a repo that Jay linked me to that said it is set when DEBUG is enabled (not production) to enable the VS Code debugger to break on raised exceptions. Just copied the line from that repo in the next push.